packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -161,14 +211,30 @@ export class ERC1155Standard {
     standard: string;
     tokenURI: string | undefined;
     image: string | undefined;
+    name: string | undefined;
+    symbol: string | undefined;
   }> => {
     const isERC1155 = await this.contractSupportsBase1155Interface(address);
 
     if (!isERC1155) {
       throw new Error("This isn't a valid ERC1155 contract");
     }
+
     let tokenURI, image;
 
+    const [getNameResult, getSymbolResult] = await Promise.allSettled([
+      this.getAssetName(address),
+      this.getAssetSymbol(address),
+    ]);
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
what type error are we ignoring here?

Reply:
we're getting the following error:
`
Property 'value' does not exist on type 'PromiseSettledResult<string>'.
  Property 'value' does not exist on type 'PromiseRejectedResult'.
`
so the linter is complaining

Reply:
ah right ok. Sorry to nit alot on this but I think we should add a comment explaining that the value is only defined on the result when successfully fulfilled. Otherwise it will be undefined which is fine in this case.

Reply:
To avoid the warning, `.status === "fulfilled"` could be checked on the promise.  

Or to be less verbose, `safelyExecute` could be used with `Promise.all` to achieve a similar effect to `Promise.allSettled`:

```
const [name, symbol] = Promise.all([
  safelyExecute(() => this.getAssetName(address)),
  safelyExecute(() => this.getAssetSymbol(address)),
]);
```

Reply:
@adonesky1 comment added , you can tell if it looks good to you then i'll resolve this conversation

Reply:
@bergeron done , i also included the execution of the `tokenURI()`  func :)